client/src/components/not-enough.tsx

```diff
@@ -17,27 +41,15 @@ export default function NotEnough({ isEnough }: { isEnough: boolean }) {
           <div className="w-full h-full flex flex-col items-center justify-center space-y-5">
             <h3 className="text-white font-semibold text-2xl">Missing Token</h3>
             <p className="text-center text-white text-lg font-medium">
-              This Game is currently available to the general public, Input an
-              email address to be notified when available to all users
+              This Game is currently available to Mancala Holder, Click the mint token button.
```

Comment:
Change this to

"This game is currently available to Mancala Pass NFT holders, click the "Mint Mancala Pass" button to mint a testnet pass.